frontend/src/components/common/Resource/CreateButton.tsx

```diff
@@ -15,42 +15,53 @@ export default function CreateButton() {
   const location = useLocation();
   const { t } = useTranslation(['resource', 'frequent']);
 
-  const applyFunc = async (newItem: KubeObjectInterface) => {
-    try {
-      await apply(newItem);
-    } catch (err) {
-      let msg = t('Something went wrong…');
-      if (err instanceof Error) {
-        msg = err.message;
-      }
-      setErrorMessage(msg);
-      setOpenDialog(true);
-      throw err;
-    }
+  const applyFunc = async (newItems: KubeObjectInterface[]) => {
+    await Promise.allSettled(newItems.map(newItem => apply(newItem))).then((values: any) => {
+      values.forEach((value: any, index: number) => {
+        if (value.status === 'rejected') {
+          let msg = t('Something went wrong…');
```

Comment:
This value is always replaced below, so it will never be used AFAICS.

Reply:
changed the msg to be appended with this string now